dds_cli/text_handler.py

```diff
@@ -7,6 +7,7 @@
 # Standard library
 import logging
 import pathlib
+from dds_cli import dds_on_legacy_console
```

Comment:
Change to `import dds_cli` and use as `dds_cli.dds_on_legacy_console` instead. 
Also add `dds_on_legacy_console` to the `__all__` list in `__init__.py`

Reply:
Thanks for the fast review. I changed what you requested. On the current dev branch we don't have an `_all_` list yet and also on the other branches that I checked (such as your dev) I couldn't find one.  So I created that list with all the symbols included. I hope this was what you intended to do.

Reply:
Yes `__all__ = ['DDSEndpoint','FileSegment','dds_questionary_styles', 'dds_on_legacy_console']` was exactly what I meant!